collections/harmonized-landsat-sentinel.yaml

```diff
@@ -0,0 +1,526 @@
+      common_name: rededge
+      center_wavelength: 0.7405
+      full_width_half_max: 0.015
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: RedEdge3
+      common_name: rededge
+      center_wavelength: 0.7828
+      full_width_half_max: 0.02
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: NIR_Broad
+      common_name: nir
+      center_wavelength: 0.8328
+      full_width_half_max: 0.106
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: NIR_Narrow
+      common_name: nir08
+      center_wavelength: 0.86467
+      full_width_half_max: 0.02825
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: SWIR1
+      common_name: swir16
+      center_wavelength: 1.60886
+      full_width_half_max: 0.08472
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: SWIR2
+      common_name: swir22
+      center_wavelength: 2.20073
+      full_width_half_max: 0.18666
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: WaterVapor
+      common_name: nir09
+      center_wavelength: 0.9451
+      full_width_half_max: 0.02
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: Cirrus
+      common_name: cirrus
+      center_wavelength: 1.37343
+      full_width_half_max: 0.02039
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: Thermal Infrared 1
+      common_name: lwir11
+      center_wavelength: 10.895
+      full_width_half_max: 0.8
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - name: Thermal Infrared 2
+      common_name: lwir12
+      center_wavelength: 12.005
+      full_width_half_max: 1
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - description: Quality Assessment band (QA)
+      name: QA
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - description: View (sensor) Azimuth Angle
+      name: VAA
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - description: View (sensor) Zenith Angle
+      name: VZA
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - description: Sun Azimuth Angle
+      name: SAA
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - description: Sun Zenith Angle
+      name: SZA
+      openeo:gsd:
+        value:
+          - 30
+          - 30
+        unit: m
+    - description: The mask of data/no data pixels.
+      name: dataMask
+  constellation:
+    - SENTINEL
+    - LANDSAT
+  gsd:
+    - 10
+    - 20
+    - 60
+  instruments:
+    - oli
+    - msi
+  eo:cloud_cover:
+    minimum: 0
+    maximum: 100
+  platform:
```

Comment:
I had a short talk on this with Matthias and it's probably better to not list this here to avoid confusion. Each item in the collection can be traced back to one of the platforms but not the other way around. Same goes for instruments and constellation.

Reply:
Removed `platform`, `instruments` and `constellation`.